best_practices/workflows_misc.md

```diff
@@ -0,0 +1,29 @@
+# Workflows - Miscellaneous
+
+
+## Citing
```

Comment:
Citation information is embedded everywhere, in the Galaxy workflow, in dockstore/workflowhub etc, I don't think this is necessary to repeat here.